armory/__about__.py

```diff
@@ -0,0 +1 @@
+VCS_VERSION = "0.15.5.dev83+g2906e99.d20220921"
```

Comment:
Hard coding this seems to go completely against the reason for SCM versioning.

Reply:
How will this update when a new commit is added?

Reply:
I [replied here](https://github.com/twosixlabs/armory/pull/1617#discussion_r976677381) as the comments are in a similar vein.